src/librustc_middle/mir/interpret/allocation.rs

```diff
@@ -162,9 +162,9 @@ impl<Tag, Extra> Allocation<Tag, Extra> {
         &self.bytes[range]
     }
 
-    /// Returns the undef mask.
-    pub fn undef_mask(&self) -> &UndefMask {
-        &self.undef_mask
+    /// Returns the uninit mask.
```

Comment:
```suggestion
    /// Returns the mask indicating which bytes are initialized.
```

Reply:
Fixed in last commit.

Reply:
Is it? The latest diff still shows the old comment.